src/command_modules/azure-cli-shell/azclishell/util.py

```diff
@@ -89,3 +89,11 @@ def parse_quotes(cmd, quotes=True, string=True):
         return str_args
     else:
         return args
+
+
+def get_os_clear_screen_word():
+    """ keyword to clear the screen """
+    if platform.system() == 'Windows':
+        return 'cls'
```

Comment:
"cls" isn't so much Windows specific as it is "cmd.exe" specific. If you try this on bash on Windows and it requires cls then that would be a bug, especially if clear didn't work! Then again platform.system() might resolve to Linux when run in bash on Windows and in that case you'd be fine. Please check that out before you merge.